IFC4x3/Constants/l/LINE_0aQELtHxv8tPRsAoCd1LjP.xml

```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="LINE_0aQELtHxv8tPRsAoCd1LjP" Name="LINE" UniqueId="2468e577-47be-48dd-96f6-2b2327055b59">
-	<Documentation>Straight connection between two points.</Documentation>
+	<Documentation>In the geometry perspective it denotes a straight connection between two points. In the dynamic perspective, it denotes a segment with a curvature with a value of 0. This means that no lateral acceleration acts on the moving train or vehicle.
```

Comment:
Is a train not a vehicle?

Reply:
The term "train" might be a little bit vague here. 

Most commonly it is used to denote an assembly of vehicles which from rail operations view (e.g. planning, dispatching, securing safe movement, accounting) act as a (temporary) unit. In common rail the vehicles are categorized into locomotives and wagons, the locomotives sometimes also carrying passengers or freight (railcar).

@SergejMuhic : do we need definitions of this kind? If yes, should it be placed near the usage of the term in type and entitiy definitions or should this go into chapter 3 "Terms, definitions, and abbreviated terms"?

Reply:
Concerning Cubic: in the literature it is almost always discussed with the 3rd order term only. The main problem here is that y is a function of x,  whereas in the other transition types, both x and y are a function of s.

I think that indeed it is OK to state the full polynomial in the definition. The documentation should point out the 3rd order term only aspect and also explain the y=f(x) versus y=f(s) aspect. Here one could also mention engineering shortcuts like approximating x with s, known in some parts of the world as the cubic spiral.

@SergejMuhic OK?

Reply:
Regarding train/vehicle, the shared definition says it already 

> Most commonly it is used to denote an assembly of vehicles

so, I would remove train and just leave vehicle as it is a hypernym of train.

Reply:
"Train" will be removed,

Concerning CUBICSPIRAL I hope to get a final advice from a top track engineering expert today. --> I propose to isolate removal of CUBICSPIRAL and documentation into a separate small PR.